### PR TITLE
Swedbank Pay Checkout updates and fixes

### PR DESCRIPTION
**Pull request by aait**

### CODE REVIEW

**Review by olegisk — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -34,11 +34,18 @@ install:
   - cd woocommerce/
   - git checkout tags/$WOO_VERSION
   - bash tests/bin/install.sh woocommerce_test root '' localhost $WP_VERSION
+  - cd /tmp
+  - mkdir /tmp/tmp1
+  - wget https://github.com/SwedbankPay/swedbank-pay-woocommerce-payments/archive/master.zip
```
Comment (asbjornu) [inline .travis.yml:39]: Can't `swedbank-pay-woocommerce-payments` be installed with composer instead?

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (aait) [reply .travis.yml:39]: Screenshot

Comment (asbjornu) [reply .travis.yml:39]: Yes, but can't we get that published on Packagist first, then add it as a Composer requirement here after?

Comment (aait) [reply .travis.yml:39]: Yes, will do

File: templates/checkout/payex/checkin.php [hunk 2]
```diff
@@ -27,12 +28,8 @@
...
 </div>
 
-<?php if ($consumer_profile): ?>
-	<div id="payex-consumer-profile" data-reference="<?php esc_html_e( $consumer_profile ); ?>"></div>
-<?php endif; ?>
-
 <div style="clear: both;">&nbsp;</div>
 
-<button id="change-shipping-info" type="button" class="button" style="display: none;">
+<!-- <button id="change-shipping-info" type="button" class="button" style="display: none;">
```
Comment (asbjornu) [inline templates/checkout/payex/checkin.php]: Why not remove the button entirely?

File: templates/checkout/payex/payment.php [hunk 3]
```diff
@@ -1,7 +1,16 @@
...
+
+/** @var string $js_url */
 ?>
 <!-- <button type="submit" class="button alt" name="woocommerce_checkout_place_order" id="place_order" value="Place order" data-value="Place order">Place order</button> -->
 <h3>3. <?php _e( 'Payment', 'woocommerce-gateway-payex-checkout' ); ?></h3>
 <div id="payment-payex-checkout"></div>
+<?php if ($js_url): ?>
+	<script>
+        window.onload = function () {
+            //wc_payex_checkout.initPaymentJS( '<?php echo $js_url; ?>' );
```
Comment (asbjornu) [inline templates/checkout/payex/payment.php]: What's the point of this `onload` function?

**Review by asbjornu — CHANGES_REQUESTED** [review]

**Review by asbjornu — APPROVED** [review]

**Comment by asbjornu** [comment]
Can you please rename everything "PayEx" to "Swedbank Pay"?

**Pull request merged**